src/varpubs/pubmed_db.py

```diff
@@ -14,27 +24,193 @@ class PubmedArticle(SQLModel, table=True):
     doi: str
 
 
+class TermToPMID(SQLModel, table=True):
+    """Mapping table between HGVS.p terms and PubMed PMIDs."""
+    term: str = Field(nullable=False)
+    pmid: int = Field(Integer, primary_key=True, nullable=False)
+    __table_args__ = (sqlalchemy.PrimaryKeyConstraint("term", "pmid"),)
+
+
 @dataclass
 class PubmedDB:
+    """
+    Class responsible for building and managing a local DuckDB database
+    that maps HGVS.p variant annotations to PubMed articles.
+    """
     path: Path
-    _engine: sqlalchemy.engine.base.Engine | None = field(init=False)
+    vcf_paths: Iterable[Path]
+    email: str
+    _engine: Optional[sqlalchemy.engine.base.Engine] = field(init=False, default=None)
 
     def deploy(self) -> None:
-        # wipe existing database
-        if self.path.exists():
-            self.path.unlink()
-        # create parent directories if they don't exist
+        """
+        Main pipeline entry point. Extracts HGVS.p terms from VCFs,
```

Comment:
Don't think the information of it being the entry point adds substantial information here so we can just leave it out.
```suggestion
        Extracts HGVS.p terms from VCFs,
```